modules/common/src/utils/utils.ts

```diff
@@ -37,6 +38,7 @@ export function getPropertyOrUndefined<T> (property: T | null | undefined): T |
 }
 
 export function getTrueOrUndefined (value: boolean): true | undefined {
+    // eslint-disable-next-line unicorn/no-negated-condition, unicorn/prefer-logical-operator-over-ternary
     return !value ? undefined : value;
```

Comment:
why didnt you turn it around here like everywhere else?

Reply:
because here empty string is possible value and doesn't work with `??`